bitnami/postgresql/templates/statefulset.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apps/v1beta1
```

Comment:
I see different solutions for the StatefulSet API version.. Some charts use `apps/v1beta1`, others `apps/v1beta2` and others `apps/v1`. We should standardise it.

Reply:
You are right! I think we should use `apps/v1beta2`. 
The `apps/v1` only works in very new kubernetes clusters and we have already had issues in the past with this API version. https://github.com/kubernetes/charts/issues/4908